testing/jormungandr-integration-tests/src/jormungandr/mempool.rs

```diff
@@ -438,3 +439,71 @@ pub fn expired_fragment_should_be_rejected() {
     FragmentVerifier::wait_and_verify_is_rejected(Duration::from_secs(1), check, &jormungandr)
         .unwrap();
 }
+
+#[test]
+/// Verifies that a passive node will reject a fragment that has expired, even after it's been
+/// accepted in its mempool.
+fn expired_fragment_should_be_rejected_by_passive_node() {
+    const N_FRAGMENTS: u32 = 10;
+
+    let receiver = startup::create_new_account_address();
+    let mut sender = startup::create_new_account_address();
+
+    let (leader, _) = startup::start_stake_pool(
```

Comment:
One remark. I see that you are using GenesisPraos consensus (start_stake_pool). Basically we support also BFT, if you change it to bft consensum mayb we would have better matrxi coverage of passive/leader vs bft/GPraos coverage

Reply:
Should we add one test with Praos + Leader and one with BFT + passive? Or 4 tests to cover all pairs?